data/presentations.yaml

```diff
@@ -1,3 +1,146 @@
+- title: "Community Introductions: Ed Warnicke at Network Service Mesh"
```

Comment:
I'd suggest making these titles a little punchier. eg. "How Network Service Mesh uses SPIFFE and SPIRE"